packages/jest-config/src/normalize.js

```diff
@@ -200,7 +200,9 @@ const normalizeCollectCoverageFrom = (options: InitialOptions, key: string) => {
       value = JSON.parse(options[key]);
     } catch (e) {}
 
-    Array.isArray(value) || (value = [options[key]]);
+    if (value && !Array.isArray(value)) {
```

Comment:
it ended up being `[null]`, which threw below in the loop.

Also, super weird to use `||` then assignment...